tests/interpreters/test_onnx_interpreter.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+
+from xdsl.dialects import onnx
+from xdsl.dialects.builtin import ModuleOp, TensorType, f32
+from xdsl.interpreter import Interpreter
+from xdsl.interpreters.shaped_array import ShapedArray
+from xdsl.utils.test_value import TestSSAValue
+
+try:
+    import numpy as np
+except (ImportError, ModuleNotFoundError):
+    np = None
+    pytest.importorskip("numpy", reason="numpy is an optional dependency in xDSL")
```

Comment:
```suggestion
pytest.importorskip("numpy", reason="numpy is an optional dependency in xDSL")
```
does this work?

Reply:
nope

Reply:
what's the error?

Reply:
`  import numpy as np
E   ModuleNotFoundError: No module named 'numpy'` 
 this is the error